recipes-connectivity/networkmanager/networkmanager_1.18.4.bbappend

```diff
@@ -0,0 +1,55 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}-${PV}:${THISDIR}/files:"
+
+SRC_URI += " \
+    file://fix-compatibility-with-network-slave.patch \
+    file://always-assume-uid-for-external-processes.patch \
+    file://auto-dhcp.patch \
+    file://db-nm-settings.patch \
+    file://fix-eth0-slave-issues.patch \
+    file://update-resolv-conf.patch \
+    file://update-routing-tables.patch \
+    file://use-dom0-db-for-seen-bssids.patch \
+    file://disable-ipv6-config.patch \
+    file://fix-network-reenable.patch \
+    file://NetworkManager.conf \
+    file://nm_sync.sh \
+    file://db_to_nm.awk \
+    file://nm_to_db.awk \
+    file://org.openxt.nmapplet.conf \
+    file://dbus-system.conf \
+"
+
+do_install_prepend () {
+    sed -i -e s:deny:allow:g ${S}/src/org.freedesktop.NetworkManager.conf
+    sed -i -e s:deny:allow:g ${S}/dispatcher/nm-dispatcher.conf
```

Comment:
This is terrifying. From what I see it enables the following from `org.freedesktop.NetworkManager.conf`:
```xml
        <policy context="default">
```
```xml
                <deny own="org.freedesktop.NetworkManager"/>
                <deny send_destination="org.freedesktop.NetworkManager"/>
```
```xml
                <!-- Root-only functions -->
                <deny send_destination="org.freedesktop.NetworkManager"
                      send_interface="org.freedesktop.NetworkManager"
                      send_member="SetLogging"/>
                <deny send_destination="org.freedesktop.NetworkManager"
                      send_interface="org.freedesktop.NetworkManager"
                      send_member="Sleep"/>
                <deny send_destination="org.freedesktop.NetworkManager"
                      send_interface="org.freedesktop.NetworkManager.Settings"
                      send_member="LoadConnections"/>
                <deny send_destination="org.freedesktop.NetworkManager"
                      send_interface="org.freedesktop.NetworkManager.Settings"
                      send_member="ReloadConnections"/>

                <deny own="org.freedesktop.NetworkManager.dnsmasq"/>
                <deny send_destination="org.freedesktop.NetworkManager.dnsmasq"/>
```

[nm-dispatcher.conf](https://github.com/NetworkManager/NetworkManager/blob/1.18.4/dispatcher/nm-dispatcher.xml) does not have `deny` statement though.

Reply:
Yes, this is carried as legacy even in 1.4.4. But this is no longer required, I will remove it.
Thanks.